### PR TITLE
Update docs and external client for priority API change

### PR DESCRIPTION
**Pull request by Anirudh-Scale**
[[sc-724838]](https://app.shortcut.com/scaleai/story/724838)

### CODE REVIEW

**Comment by Anirudh-Scale** [comment]
Screenshot 2023-03-21 at 3 25 35 PM
Screenshot 2023-03-21 at 3 25 44 PM
Screenshot 2023-03-21 at 3 25 51 PM
Screenshot 2023-03-21 at 3 25 59 PM

File: launch/client.py [hunk 1]
```diff
@@ -740,6 +744,7 @@ def create_model_endpoint(
                 max_workers=max_workers,
                 per_worker=per_worker,
                 gpu_type=gpu_type,
+                high_priority=high_priority,
```
Comment (phil-scale) [inline launch/client.py:747]: In this case, I think you'll want to change `CreateModelEndpointRequest` to `CreateModelEndpointRequestV1` in line 805. I think you'll get an error if you try to create a `CreateModelEndpointRequest` as is

File: launch/client.py [hunk 2]
```diff
@@ -956,6 +966,7 @@ def edit_model_endpoint(
                 min_workers=min_workers,
                 model_bundle_id=model_bundle_id,
                 per_worker=per_worker,
+                high_priority=high_priority,
```
Comment (phil-scale) [inline launch/client.py:969]: Same thing here, with `UpdateModelEndpointRequest` to `UpdateModelEndpointRequestV1`

**Review by phil-scale — APPROVED** [review]
LGTM! To deploy the updated docs to the external site, run `mkdocs gh-deploy` whenever you're ready

**Pull request merged**